environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - python
   - python-dateutil
   - scipy>=1.4.0
-  - spiceypy>=2.3.0
+  - spiceypy>=4.0.1
```

Comment:
ncols, nrows was deprecated in 4.0.1 https://spiceypy.readthedocs.io/en/main/changelog.html#id5